doc/onboarding.md

```diff
@@ -108,7 +108,7 @@ onboarding session.
   * what belongs in node:
     * opinions vary, but I find the following helpful:
     * if node itself needs it (due to historic reasons), then it belongs in node
-      * that is to say, url is there because of http, freelist is there because of http, et al
+      * that is to say, URL is there because of HTTP, free list is there because of HTTP, et al
```

Comment:
All of the changes here are incorrect. They are actually referring to built-in modules (internal or otherwise). If anything, backticks should simply be added to help highlight this fact.